python-backend/app/api/nris_services/NRIS_service.py

```diff
@@ -40,35 +45,40 @@ def get_EMPR_data_from_NRIS(mine_no):
 
     try:
         token = get_NRIS_token()
-    except requests.exceptions.HTTPError:
+    except:
         raise
 
     if token is None:
         return None
+    
+    url = current_app.config['NRIS_INSPECTION_URL']
 
-    #Inspection start date is set to 2018-01-01 as that is the begining of time for NRIS
-    params = {
-        'inspectionStartDate':
-        '2018-01-01',
-        'inspectionEndDate':
-        f'{current_date.year}-{current_date.month}-{current_date.day}',
-        'mineNumber':
-        mine_no,
-    }
+    if url is None:
+        raise TypeError('Could not load the NRIS URL.')
+    else:
+        #Inspection start date is set to 2018-01-01 as that is the begining of time for NRIS
```

Comment:
Debatable, but you could assign the date to a descriptive variable instead of this comment